r/R/csv.R

```diff
@@ -623,9 +638,58 @@ readr_to_csv_convert_options <- function(na,
 #' @include arrow-package.R
 write_csv_arrow <- function(x,
                             sink,
+                            file = NULL,
                             include_header = TRUE,
-                            batch_size = 1024L) {
-  write_options <- CsvWriteOptions$create(include_header, batch_size)
+                            col_names = NULL,
+                            batch_size = 1024L,
+                            write_options = NULL,
+                            ...) {
+  unsupported_passed_args <- names(list(...))
+
+  if (length(unsupported_passed_args)) {
+    stop(
+      "The following ",
+      ngettext(length(unsupported_passed_args), "argument is ", "arguments are "),
+      "not yet supported in Arrow: ",
+      oxford_paste(unsupported_passed_args),
+      call. = FALSE
+    )
+  }
+
+  if (!missing(file) && !missing(sink)) {
+    stop(
+      paste("You have supplied both \"file\" and \"sink\" arguments. Please",
+            "supply only one of them."),
+      call. = FALSE
+    )
```

Comment:
Can simplify this by removing the `paste()` since `stop()` already pastes together its arguments (with no separator)
```suggestion
    stop(
      "You have supplied both \"file\" and \"sink\" arguments. ",
      "Please supply only one of them.",
      call. = FALSE
    )
```

Reply:
Done